spec/form_models/publishers/job_listing/important_dates_form_spec.rb

```diff
@@ -0,0 +1,202 @@
+require "rails_helper"
+
+RSpec.describe Publishers::JobListing::ImportantDatesForm, type: :model do
+  subject { described_class.new(params) }
+
+  context "validations" do
+    let(:params) { {} }
+
+    it { should validate_presence_of(:publish_on).with_message(I18n.t("important_dates_errors.publish_on.blank")) }
```

Comment:
We should be consistent everywhere and either use `should` or `is_expected.to`. I prefer the latter, but if the team agrees on the former, I would be happy too